src/Canvas.js

```diff
@@ -1,19 +1,18 @@
-
-import React, { useState, useCallback}  from 'react';
-import FreeForm from './FreeformIdeas';
-import Options from './OptionBar';
-
+import React, { useState, useCallback } from "react";
+import FreeForm from "./FreeformIdeas";
+import Options from "./OptionBar";
 
 const Window = () => {
-    const [hideSourceOnDrag, setHideSrouceOnDrag] = useState(true);
-    const toggle = useCallback(() => setHideSrouceOnDrag(!hideSourceOnDrag), [
-        hideSourceOnDrag,
-    ]);
-    return (<div className="Containment">
-            <FreeForm id="freeFormMap" hideSourceOnDrag={hideSourceOnDrag}/>
-            <Options/>
-    </div>);
+  const [hideSourceOnDrag, setHideSrouceOnDrag] = useState(true);
+  const toggle = useCallback(() => setHideSrouceOnDrag(!hideSourceOnDrag), [
+    hideSourceOnDrag,
+  ]);
```

Comment:
We've removed `hideSourceOnDrag`, right?

Reply:
Yes!
I used control + F to look through the documents but I forgot that "Canvas" was a thing..